utils/manifests/error.go

```diff
@@ -11,12 +15,15 @@ const (
 	ErrAbsentFilterCode          = "meshkit-11238"
 	ErrCreatingDirectoryCode     = "meshkit-11239"
 	ErrGetResourceIdentifierCode = "meshkit-11240"
+	ErrYamlUnmarshalSyntaxCode   = "replace_me"
 )
 
 func ErrGetResourceIdentifier(err error) error {
 	return errors.New(ErrGetResourceIdentifierCode, errors.Alert, []string{"Error extracting the resource identifier name"}, []string{err.Error()}, []string{"Could not extract the value with the given filter configuration"}, []string{"Make sure to input a valid manifest", "Make sure to provide the right filter configurations", "Make sure the filters are appropriate for the given manifest"})
 }
-
+func ErrYamlUnmarshalSyntax(err string, packageName string) error {
+	return errors.New(ErrYamlUnmarshalSyntaxCode, errors.Alert, []string{"Unmarshal syntax error."}, []string{fmt.Sprintf("helm chart of %s has syntax error(s): \n%s", packageName, err)}, []string{"Invalid object format"}, []string{"Make sure to input a valid YAML object"})
```

Comment:
Why are you being specific about the helm chart in this general yaml unmarshal?

Reply:
I believe this error is no longer necessary since we have resolved the issues with the current model and error naming. Now, errors for a model are displayed directly under its name, making this pull request redundant.

Regarding the specific file mentioned by Lee, @MUzairS15 , I am unable to find a way to implement it. This is because our manifest contains all the files, and if there is an issue with the metadata, the name is not retrieved.